packages/xo-server-audit/src/index.js

```diff
@@ -135,6 +136,20 @@ class AuditXoPlugin {
     this._cleaners = []
     this._xo = xo
 
+    const {
+      enabled = true,
+      schedule = {
+        cron: '0 6 * * *',
+      },
+    } = staticConfig.lastHashUpload ?? {}
+
+    if (enabled) {
+      this._uploadLastHashJob = createSchedule(
+        schedule.cron,
```

Comment:
Ping

Reply:
Sorry, i don't understand what do you mean.

Reply:
My bad